docs/dashboards/heatmap.md

```diff
@@ -0,0 +1,21 @@
+title: Heatmap
+description: Shows a single data series in a 24x7 heatmap
+
+The Heatmap is a two-dimensional representation of data, where values are represented using color saturation. The vertical axis shows weekdays, while the horizontal axis represents the hours in a day.
```

Comment:
Would it be true to say that our Heatmap is *specifically* for fields that contain time and it will always show the time-of-the-day + day-of-the-week on the two axes?

Reply:
It is not specifically for fields that contain time, for example if you go to any Logs or Monitoring report and then convert an existing bar, pie, timeseries etc chart to heatmap and it will work.

Reply:
OK